qemu/tests/cfg/in_place_upgrade.cfg

```diff
@@ -0,0 +1,81 @@
+- in_place_upgrade:
+    pre_release = "release 8"
+    post_release = "release 9"
+    type = in_place_upgrade
+    login_timeout = 300
+    release_check = "cat /etc/redhat-release"
+    new_kernel_ver = "5.14"
+    fix_firewalld = "sed -i 's/AllowZoneDrifting=yes/AllowZoneDrifting=no/g' "
+    fix_firewalld += "/etc/firewalld/firewalld.conf"
+    fix_permit = "sed -i 's/PermitRootLogin yes/PermitRootLogin no/g' "
+    fix_permit += "/etc/ssh/sshd_config"
+    restore_permit = "sudo sed -i 's/PermitRootLogin no/PermitRootLogin yes/g'"
+    restore_permit += " /etc/ssh/sshd_config"
+    check_file = "sudo ls /var/log/leapp/answerfile"
+    restart_sshd = "sudo systemctl restart sshd.service"
+    add_ipuser = "useradd ipuser"
+    add_wheel_ipuser = "usermod -aG wheel ipuser"
+    add_passwd_ipuser = "echo ipuser |passwd --stdin ipuser"
+    no_passwd_for_sudo = "echo -e 'ipuser\tALL=(ALL)\tNOPASSWD: ALL' > "
+    no_passwd_for_sudo += "/etc/sudoers.d/leapp_ipuser_sudo"
+    check_kernel = "uname -r"
+    ipu_after_timeout = 6000
+    user_assistant = "ipuser"
+    user_assistant_pw = "ipuser"
+    reboot_cmd = "reboot"
+    yum_update = "yum -y update"
+    clean_up_old_kernel = "dnf -y remove --oldinstallonly"
+    depends_pkgs = "yum-utils"
+    error_info = "Error: No old installonly packages found for removal"
+    stop_yum_update = "rm -rf /var/run/yum.pid"
+    check_repo_list = "yum repolist enabled"
+    variants:
+        - without_rhsm_rhel8_to_rhel9:
+            rhsm_type = "no_rhsm"
+            pre_upgrade_no_rhsm = "leapp preupgrade --debug --no-rhsm"
+            process_upgrade_no_rhsm = "leapp upgrade --debug --no-rhsm"
+            disable_content = "yum-config-manager --disable "*""
+            x86_64:
+                enable_content = "yum-config-manager"
+                enable_content += " --enable rhel-8-for-x86_64-appstream-rpms"
+                enable_content += " --enable rhel-8-for-x86_64-baseos-rpms"
+            ppc64, ppc64le:
+                enable_content = "yum-config-manager"
+                enable_content += " --enable rhel-8-for-ppc64le-appstream-rpms"
+                enable_content += " --enable rhel-8-for-ppc64le-baseos-rpms"
+            s390x:
+                enable_content = "yum-config-manager"
+                enable_content += " --enable rhel-8-for-s390x-appstream-rpms"
+                enable_content += " --enable rhel-8-for-s390x-baseos-rpms"
```

Comment:
I think can simplify to `--enable rhel-8-for-${vm_arch_name}-appstream-rpms`, `--enable rhel-8-for-${vm_arch_name}-baseos-rpms`

Reply:
good points, and will update it.

Reply:
Update this patch as we discussed, could you please help to review this patch if you are free, thanks a lot :) @PaulYuuu